### PR TITLE
My first License PR

### LINKED ISSUE #6: Add LICENSE file

### PR DESCRIPTION
**Pull request by kittyweg**
My first License PR  'fix #6'

### CODE REVIEW

**Comment by kittyweg** [comment]
> Have you addressed the review comments?

I think I did, I added the fix #6 - it is not correct?

**Comment by lioramilbaum** [comment]
I don't see it in the first PR comment. What am I missing?
There were additional review comments.

**Comment by kittyweg** [comment]
I think I know what happened. I didn't know how to add the fix you requested. I searched since Friday, found the amend option for the comment, but it didn't work for me. So I made a small change in the License file and recommitted with the comment "fix #6" in the hopes it would put it in the correct place. I understand that what you want is for my change to appear in the 6th place, but I haven't found how to do it yet. What I did is incorrect?

**Comment by lioramilbaum** [comment]
You should use the edit comment. See the attached screenshot.

Screenshot from 2020-01-05 12-41-22

**Comment by kittyweg** [comment]
@lioramilbaum I think I finished, can you please check?

File: README.md [hunk 1]
```diff
@@ -1 +1,3 @@
 # baboon
```
Comment (lioramilbaum) [inline README.md]: Why is this file included in the PR at all? Please address ALL review comments.

**Comment by kittyweg** [comment]
Because I added a line in the file.


Kitty Weg

Technical Support Engineer

Red Hat Israel <https://www.redhat.com/>

34 Jerusalem rd. Ra'anana, 43501

*kweg@redhat.com <kweg@redhat.com> *  T: *+972-9-**7692287 *
M: *+972-54-5664256*   F: +972-9-7692223
@RedHat <https://twitter.com/redhat>   Red Hat
<https://www.linkedin.com/company/red-hat>  Red Hat
<https://www.facebook.com/RedHatInc>
<https://red.ht/sig>

**Comment by kittyweg** [comment]
Sorry, it was sent by mistake, pls ignore.


Kitty Weg

Technical Support Engineer

Red Hat Israel <https://www.redhat.com/>

34 Jerusalem rd. Ra'anana, 43501

*kweg@redhat.com <kweg@redhat.com> *  T: *+972-9-**7692287 *
M: *+972-54-5664256*   F: +972-9-7692223
@RedHat <https://twitter.com/redhat>   Red Hat
<https://www.linkedin.com/company/red-hat>  Red Hat
<https://www.facebook.com/RedHatInc>
<https://red.ht/sig>

**Comment by lioramilbaum** [comment]
You can delete comments if they are sent by mistake. 
We will cover the PR today in our session.
